### PR TITLE
Note modal

### PR DESCRIPTION
**Pull request by tanishk15sharma**
### Note Modal

> Style
- Note modal
- Note label 

> Functionality
- Text editor
- Note modal State management

### CODE REVIEW

File: src/components/note-modal/NoteModal.jsx [hunk 1]
```diff
@@ -1,37 +1,99 @@
 import React, { useState } from "react";
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import { LabelModal } from "./label-modal/LabelModal";
```
Comment (VedanthB) [inline src/components/note-modal/NoteModal.jsx:4]: you can do a barrel import/export , make the code look clean

File: src/components/note-modal/label-modal/LabelModal.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
```
Comment (spraveenofficial) [inline src/components/note-modal/label-modal/LabelModal.jsx]: No need to Import React in the latest Versions.

File: src/components/note-modal/NoteModal.scss [hunk 3]
```diff
@@ -10,10 +10,14 @@
...
   .note-modal {
     background-color: $white;
-    width: 50%;
+    width: 40%;
   }
-  .note-body {
-    border: 1px solid $green;
+
+  .modal-footer {
+    // border: 1px solid red;
```
Comment (VedanthB) [inline src/components/note-modal/NoteModal.scss:17]: remove commented code, if you dont need it

**Review by spraveenofficial — COMMENTED** [review]
Everything looks good to me.

**Review by Mohitkumar33 — APPROVED** [review]
clean code bro

**Review by VedanthB — APPROVED** [review]
Good work tanishk,

things i liked
- use of sass
- good variable naming
- clean structured code 

You can merge the PR 🚀

File: src/components/note-modal/NoteModal.jsx (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/note-modal/NoteModal.jsx:4]: Thank you . they are just two files so I am not doing that .

File: src/components/note-modal/label-modal/LabelModal.jsx (hunk 2 above)
Comment (tanishk15sharma) [reply src/components/note-modal/label-modal/LabelModal.jsx]: okay

**Pull request merged**